kani-driver/src/assess/mod.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use anyhow::Result;
+
+use crate::metadata::merge_kani_metadata;
+use crate::project;
+use crate::session::KaniSession;
+
+mod table_builder;
+mod table_failure_reasons;
+mod table_promising_tests;
+mod table_unsupported_features;
+
+pub(crate) fn cargokani_assess_main(mut session: KaniSession) -> Result<()> {
```

Comment:
Diff can be hard to read with code moving, so N.B. this function isn't changed other than calling the build functions from the table modules instead of locally.